src/relion/_parser/class2D.py

```diff
@@ -76,6 +96,10 @@ def _load_job_directory(self, jobdir):
         class_numbers = self.parse_star_file("_rlnClassNumber", sdfile, info_table)
         particle_sum = self._sum_all_particles(class_numbers)
         int_particle_sum = [(int(name), value) for name, value in particle_sum.items()]
+        # something probably went wrong with file reading if this is the case
+        # return empty list and hope to recover later
+        if len(int_particle_sum) == 0:
```

Comment:
You can also use
```suggestion
        if not int_particle_sum:
```
on lists to test for zero length.